pkgs/development/python-modules/django-auth-ldap/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, ldap , django_2_2 
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "django-auth-ldap";
+  version = "1.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "72848b3b036d299114be3c6ef38b12f83f6cf1cf1696c5f92e06fe45a1b8e27b";
+  };
+
+  buildInputs = [ ldap django_2_2 ]; 
```

Comment:
python packages need to be added as `propagatedBuildInputs`

Reply:
fixed